interfaces/navigateur/public/js/app/outil/outilTableSelection.js

```diff
@@ -78,13 +86,110 @@ define(['outil', 'aide'], function(Outil, Aide) {
         } else if (this.options.type === 'zoom'){
             this.options.couche.zoomerOccurences(this.options.couche.obtenirOccurencesSelectionnees());
         } else if (this.options.type === 'auto'){
-            this.options.couche.zoomAuto = !this._bouton.checked;
+            if((!lancementManuel && !this._bouton.checked) || (lancementManuel && this._bouton.checked)){
+                this.options.couche.ajouterDeclencheur('vecteurOccurenceSelectionnee', function(e){
+                    e.target.zoomerOccurence(e.occurence);
+                }, {scope: this, id:'occurenceCliqueTableSelection'});
+            }else{
+                this.options.couche.enleverDeclencheur('vecteurOccurenceSelectionnee', 'occurenceCliqueTableSelection',function(e){
+                    e.target.zoomerOccurence(e.occurence);
+                });
+            }
+
+        }  else if (this.options.type === 'contenupage'){
+            
+            if((!lancementManuel && !this._bouton.checked) || (lancementManuel && this._bouton.checked)){
+                              
+                //ne pas permettre l'option d'afficher seulement la sélection
+                $.each(this.parent.obtenirOutils(), function(index,value){
+                    if(value.options.type === 'selectionSeulement'){
+                        value._bouton.setChecked(false);
+                        value._bouton.disable();
+                        value.options.couche.options.selectionSeulement = false;
+                    }
+                });
+               
+                $.each(this.options.couche.obtenirStyles(), function(index, value){
+                    value.reinitialiserFiltres();
+                    value.ajouterFiltre({filtre: function(occurence){
+                                                    if(occurence.vecteur.panneauTable.obtenirIndexParOccurence(occurence) == -1){
+                                                        return false;
+                                                    }
+                                                    else{
+                                                        return true;
+                                                    }  
+                                                
+                                            }, 
+                                            style: new Style({
+                                                    display:false,
+                                                    titre:'test'
+                                                })
+                                        });
+                })
+            }
+            else{
+
+                $.each(this.options.couche.obtenirStyles(), function(index, value){
```

Comment:
var style = this.options.couche.obtenirStyle('defaut');
                style.definirPropriete('visible', undefined);